examples/01-Detailed-Examples/05-mesh-exploration.py

```diff
@@ -0,0 +1,210 @@
+for k, v in mesh.named_selections.items():
+    print(f"{k} = {v}")
+
+###############################################################################
+# Get elements
+# ------------
+
+# Get an element by ID
+el_by_id = mesh.elements.by_id[1]
+
+# Get an element by index
+index = el_by_id.index
+print(mesh.elements[index])
+
+###############################################################################
+# Element Types and Materials
+# ---------------------------
+
+# Get the element types
+el_types = mesh.element_types
+print(el_types)
+
+# Get the materials
+e_materials = mesh.materials
+print(e_materials)
+
+###############################################################################
+# Query information about one particular element
+# ----------------------------------------------
+
+# Get the nodes of an element
+mesh.elements[1].nodes
+
+# Get the node IDs of an element
+mesh.elements[1].node_ids
+
+# Get the nodes of an element
+mesh.elements[1].n_nodes
+
+# Get the type of the element
+mesh.elements[1].type_info
+mesh.elements[1].type_id
+
+# Get the shape of the element
+mesh.elements[1].shape
+
+###############################################################################
+# Get the elemental connectivity
+# ------------------------------
+
+# get node indices from element index
+conn1 = mesh.conn_elem_to_node
+
+# get node IDs from element index
+conn2 = mesh.conn_elem_to_node_id
+
+el_idx_5 = mesh.elements[5]
+# get node IDS from element ID
+conn2.by_id[el_idx_5.id]
+
+###############################################################################
+# Get nodes
+# ---------
+
+# Get a node by ID
+node_by_id = mesh.nodes.by_id[1]
+
+# Get a node by index
+node_by_index = mesh.nodes[0]
+
+# Get the coordinates of all nodes
+print(mesh.coordinates)
+
+###############################################################################
+# Query information about one particular node
+# -------------------------------------------
+
+# Coordinates
+mesh.nodes[1].coordinates
+
+# Get Nodal connectivity
+conn3 = mesh.conn_node_to_elem
+conn4 = mesh.conn_node_to_elem_id
+
+print(mesh.nodes[0])
+
+# elements indices to elem_id
+print(conn3[0])
+
+# elements IDs from node index
+print(conn4[0])
+
+###############################################################################
+# Splitting into meshes
+# --------
+
+# Plot the mesh
+mesh.plot()
+
+# Split the global mesh according to mesh properties
+meshes = simulation.split_mesh_by_properties(
+    properties=[elemental_properties.material, elemental_properties.element_shape]
+)
+meshes.plot()
+
+# Split the global mesh and select meshes for specific property values
+print(meshes)
+meshes = simulation.split_mesh_by_properties(
+    properties={
+        elemental_properties.material: 1,
+        elemental_properties.element_shape: [0, 1],
+    }
+)
+
+# Mesh<index=0, num_nodes=100, num_elem=1000>
+meshes.plot()
+
+# Select a specific Mesh in the Meshes, by index
+meshes[1].plot()
```

Comment:
Again, when working by index, it might be better to illustrate it using "0" so we know it is unequivocally zero-based
```suggestion
meshes[0].plot()
```